### ISSUE TITLE
Add a link from mypy's main documentation page to the wiki page

### ISSUE THREAD

**Issue by edreamleo** [python/mypy#16092]
**Documentation**

mypy's [wiki page](https://github.com/python/mypy/wiki) contains superb documentation. There are links to this page from the [mypy-lang.org page](https://mypy-lang.org/) and from the [mypy repo page](https://github.com/python/mypy/tree/master), but not from the [mypy's main (readthedocs.io) documentation page](https://mypy.readthedocs.io/en/stable/).

Imo, the wiki page deserves a prominent link in the sidebar of mypy's main documentation page.

**Comment by crocks4123** [comment]
With respect to this issue, justing adding [https://mypy.readthedocs.io/en/stable/](url) to the [https://github.com/python/mypy/wiki](url) just in the sidebar would work right?

**Comment by edreamleo** [comment]
@crocks4123 Yes. With a word or two explaining that these are code-level docs.

**Comment by crocks4123** [comment]
Done added main code-level doc [https://mypy.readthedocs.io/en/stable/](url) to [https://github.com/python/mypy/wiki](url) under documentation heading. Kindly check and suggest if any changes needed.

**Comment by crocks4123** [comment]
I will look into asap. Sorry for the delay, meanwhile kindly visit the homepage in documentation section there you can see the newly added link.

Your Sincerely,
Chirag Devgade NMIMS MPSTME,
(Class Representative)
B.tech in Artificial Intelligence and Machine Learning 3rd Year
Email: ***@***.***
